yt/visualization/plot_modifications.py

```diff
@@ -3238,10 +3238,10 @@ def __init__(
         field_x,
         field_y,
         texture=None,
-        kernellen=50.0,
+        kernellen=50,
```

Comment:
a float never made sense here: `kernellen` represents the length of a 1D. I'm just fixing this in passing.